Changes

```diff
@@ -38,6 +38,10 @@ Working version
   Thread.wait_write, Thread.wait_read, the whole ThreadUnix module.
   (Nicolás Ojeda Bär, review by Damien Doligez)
 
+* #10896: Remove Stream, Genlex and Pervasives. Also remove legacy bigarray
+  library.
```

Comment:
Is it worth mentioning here that the `Bigarray` module is now included in the distribution, and that this only refers to the `bigarray` ocamlfind library? It's a little confusing potentially since the previous sentence refers to modules.

Reply:
Thanks, reworded for clarity.